bpf/lib/conntrack.h

```diff
@@ -205,6 +205,7 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 			ct_state->proxy_redirect = entry->proxy_redirect;
 			ct_state->from_l7lb = entry->from_l7lb;
 			ct_state->auth_required = entry->auth_required;
+			ct_state->from_tunnel = entry->from_tunnel;
```

Comment:
This isn't relevant for `CT_SERVICE` lookups, right? Maybe you could turn that into an if-else statement with below check (same for `ct_create4()`).

With all the flags in ct_state, it's getting more difficult to follow where they actually apply... :)

Reply:
Yep, you're right. Fixed.